js/ones_and_zeros.test.js

```diff
@@ -0,0 +1,10 @@
+const binaryArrayToNumber = require("./ones_and_zeros");
+
+describe("binaryArrayToNumber", () => {
+  test("When it gets array zeros and ones of should return decimal number", () => {
+    expect(binaryArrayToNumber([0, 0, 0, 1])).toBe(1);
+  });
+  test("When it gets array zeros and ones of should return decimal number", () => {
```

Comment:
This test name is the same as the previous one. If a test fails, you won't know which one it is. Please include the number in the test name so disambiguate them